### DISCUSSION TITLE
멀티플레이 기능이 작동하지 않습니다....  뭐가 문제일까요....

### DISCUSSION THREAD

**Discussion by zepzino** [naverz/zepeto-studio-kor#709] [5. Q&A]
현재 문제는 이렇습니다.. 유니티로 접속시는 접속로그 나오며 캐릭터가 로딩되나 타 기기로 접속할 시 서버에 로그가 남지 않고 마찬가지로 캐릭터가 로딩되지 않습니다. 

현재 멀티플레이 기능 구현을 위해 적용한 내용들은 하기와 같습니다.. 별도 코드도 첨부할게요.
(코드는 치자피즈님, 제페토 공식 예제 코드 두개다 적용해보았는데 똑같은 반응이라, 마지막은 제페토 공식 예제 코드로 적용하였습니다.)
![image](https://user-images.githubusercontent.com/100274420/167105377-35d40326-8e2b-4ba4-80c4-2d2a82be740e.png)

[CODE.zip](https://github.com/naverz/zepeto-studio-kor/files/8639063/CODE.zip)

1. schemas 설정 적용
<img width="404" alt="image" src="https://user-images.githubusercontent.com/100274420/167105626-3059e2a6-50ab-429c-b742-217f67794062.png">

2. index 코드 적용 (공식예제코드 적용)
3. ClientStarter 적용 (공식예제코드 적용)
4. Multiplay 스크립트 적용 (공식예제코드 적용)
5. Player 로더 코드 적용
<img width="450" alt="image" src="https://user-images.githubusercontent.com/100274420/167106382-e47f3ada-7981-47de-b05e-6ecfda5bdddf.png">

이외에 여기서 제가 놓친 부분이 있는건지... 아니면  외적인 요인이 있는건지 전혀 알 수가 없어 고민끝에 도움을 요청드립니다...
*방화벽 풀려있음..

부디... 저를 구해주세요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 월드의 기본 세팅 및 에디터 로그인이 되어있고 멀티플레이 서버가 켜져있는데도 나타나지 않는 상황일까요?😢
그랬는데도 동작하지 않는다면
새로운 프로젝트에서 아래 가이드 영상을 따라서 다시 만들어 보시기를 추천 드립니다.
https://studio.zepeto.me/guides/multiplay-tutorial

참고로 첨부해주신 Player 로더 코드에서 주석처리 된 부분이 핵심인데
멀티플레이의 경우는 서버 단에서 OnJoin시 생성하도록 되어 있어
클라이언트 단의 스크립트에서는 중복으로 생성되지 않도록 주석 처리 되어 있는 것입니다.

감사합니다.
